docs/reference/graph/explore.asciidoc

```diff
@@ -168,6 +180,7 @@ a maximum number of documents per value for that field. For example:
   "max_docs_per_value": 500
 }
 --------------------------------------------------
+======
```

Comment:
This closes the block section we created in https://github.com/elastic/elasticsearch/pull/55067/files#r406887032.
```suggestion
======
====
```